.github/workflows/deploy-to-control-plane.yml

```
@@ -39,36 +37,29 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+          ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.ref || steps.getRef.outputs.PR_REF || github.ref }}
```

actionlint (1.7.4)

property "getref" is not defined in object type {}

(expression)